docs/configuration.md

```diff
@@ -141,26 +141,15 @@ _note: `footer_content` is deprecated, but still supported. For a better experie
 
 ## Color scheme
 
-```yaml
-# Color scheme supports "light" (default) and "dark"
-color_scheme: dark
-```
+- `color_scheme`: Color scheme currently supports `nil` (default), `"auto"`, `"light"`, `"dark"` or a custom scheme that you define.
+- `enable_local_storage_theme`: Enable the local storage of the current theme
```

Comment:
I would say for a non-developer, it's unclear what local storage means here. I would use some different language